tools/type_whisperer/types.proto

```diff
@@ -16,6 +16,9 @@ message TypeDescription {
   // Does this type have a breaking change? If so, it needs to be upgraded at
   // the next major version.
   bool next_version_upgrade = 5;
+
```

Comment:
Would it be possible to add a protoxform golden proto test or API type DB test to verify everything works as expected?

I'm thinking that the API boosting tool is going to be largely agnostic to what this PR does, providing everything is correct in the API type database, which is great.

Reply:
yes added protoxform golden.